optuna/integration/skopt.py

```diff
@@ -229,6 +232,11 @@ def ask(self):
                 value = value * distribution.q + distribution.low
             if isinstance(distribution, distributions.IntUniformDistribution):
                 value = value * distribution.step + distribution.low
+            if isinstance(distribution, distributions.IntLogUniformDistribution):
+                value = int(
+                    np.round((value - distribution.low) / distribution.step) * distribution.step
+                    + distribution.low
+                )
```

Comment:
I think we need to truncate so that its value is included in `[low ,high]`.

Reply:
Thanks! I have updated it.